Conduit/Conduit.Db/Repositry/CommentsRepositry.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Conduit.Db.Repositry
+{
+    public class CommentsRepositry:ICommentsRepositry
+    {
+
+        private readonly ConduitContext _context;
+
+        public CommentsRepositry(ConduitContext context, IArticlesRepositry articlesRepositry)
+        {
+            _context = context;
+        }
+
+        public async Task<bool> CreateComment(Comment comment,Guid UserId)
+        {
+
+            await _context.Comments.AddAsync(
+
+                  new Comment
```

Comment:
why no using mapper?

Reply:
because CommentD not include the id of the user, so when I insert the comment it cause null because of mapping so that I did it stuff like this